src/shared_utilities/drivers.jl

```diff
@@ -1095,6 +1095,8 @@ end
 A helper function which constructs the `PrescribedAtmosphere` and `PrescribedRadiativeFluxes`
 from a file path pointing to the ERA5 data in a netcdf file, the surface_space, the start date,
 and the earth_param_set.
+
+The argument `era5_ncdata_path` is either a list of nc files, each with all of the variables required, but with different time intervals in the different files, or else it is a single file with all the variables and all of the time required. In the latter case, we convert it to a list of a single file in order to keep the same interface with TimeVaryingInput.
```

Comment:
```suggestion
The argument `era5_ncdata_path` is either a list of nc files, each with all of the variables required, but with different time intervals in the different files, or else it is a single file with all the variables and all of the time required. 
```
We don't need to discuss the implementation details in the docstring